docs/inputs/index.md

```diff
@@ -32,6 +32,7 @@ provided, a dropdown field will appear on the Inputs page. In contrast, a button
 | options                                                               | object | This property allows you to enable the [saveValidator](../advanced/save_validator.md) feature.                                                                                                                |
 | hook                                                                  | object | It is used to add custom behaviour to forms. Visit the [Custom Hook](../custom_ui_extensions/custom_hook.md) page to learn more.                                                                              |
 | [warning](../advanced/custom_warning.md)                              | object | It is used to add the custom warning message for each of the modes of 'create', 'edit', 'config', and 'clone'. The message is displayed on the form.                                                          |
+| inputHelperModule                                                     | string | A module that contains `validate_input` and `stream_events` methods. Default is `{input_name}_helper`.                                                                                                        |
```

Comment:
Can we document details about this helper class, having the following information:
- location of the helper class
- the two functions it _must_ contain with the arguments, as our framework imports both for modinput script generation.

Reply:
I updated docs, added a separate page.